econml/utilities.py

```diff
@@ -1871,3 +1872,42 @@ def fit(self, X, Y):
     def predict(self, X):
         predictions = self.model.predict(X)
         return reshape(predictions, (-1, 1)) if self.needs_unravel else predictions
+
+
+class WeightedLassoCVWrapper:
+    """Helper class to wrap either WeightedLassoCV or WeightedMultiTaskLassoCV depending on the shape of the target."""
+
+    def __init__(self, *args, **kwargs):
+        self.args = args
+        self.kwargs = kwargs
+
+    def fit(self, X, y):
```

Comment:
SUPER DUPPER IMPORTANT: add weights!!!

Reply:
Thanks for pointing it out. I check the test_dml to test 'auto' argument but didn't catch the sample_weight, will add test under test_can_use_sample_weights.